packages/forms/src/styled/tiles/StyledTile.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { getColor, DEFAULT_THEME, retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import { rgba } from 'polished';
+
+export const StyledTileInput = styled.input`
+  position: absolute;
+  border: 0;
+  clip: rect(1px, 1px, 1px, 1px);
+  padding: 0;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  white-space: nowrap;
+`;
+
+StyledTileInput.defaultProps = {
+  theme: DEFAULT_THEME
+};
```

Comment:
Would prefer to see this in its own file with the rules for view components applied:
- `data-garden-id` & `data-garden-package` attrs
- theming override capabilities via `retrieveComponentStyles`

Reply:
This is a hidden input that is not visually show and only included for a11y/interaction. I'm not sure why this would need to be themed?

Reply:
I still think the clearer pattern is to separate styled components, and to not assert whether there is theming benefit or not. But YMMV – I'll go with your preference here.

Reply:
I've moved this to a standalone file to separate them.